content/master/concepts/packages.md

```diff
@@ -275,6 +275,32 @@ spec:
 # Removed for brevity
 ```
 
+#### Automatically update dependency versions
+
+Crossplane can automatically upgrade a package's dependency version to the minimum
+valid version that satisfies all the constraints. It's an alpha feature that
+requires enabling with the `--enable-dependency-version-upgrades` flag.
```

Comment:
should we link here to help on how to set feature flags?

Reply:
done